parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelEoSStreamProcessor.java

```diff
@@ -98,8 +105,7 @@ public void pollAndProduce(Function<PollContext<K, V>, ProducerRecord<K, V>> use
 
     @Override
     @SneakyThrows
-    public void pollAndProduce(Function<PollContext<K, V>, ProducerRecord<K, V>> userFunction,
-                               Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
+    public void pollAndProduce(Function<PollContext<K, V>, ProducerRecord<K, V>> userFunction, Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
```

Comment:
```suggestion
    public void pollAndProduce(Function<PollContext<K, V>, ProducerRecord<K, V>> userFunction, 
    Consumer<ConsumeProduceResult<K, V, K, V>> callback) {
```
and run formatter, as above